src/libraries/System.Text.Encodings.Web/tests/JavaScriptStringEncoderTests.Relaxed.cs

```diff
@@ -26,23 +25,23 @@ public void TestSurrogate_Relaxed()
         public void Relaxed_EquivalentToAll_WithExceptions()
         {
             // Arrange
-            JavaScriptStringEncoder controlEncoder = new JavaScriptStringEncoder(UnicodeRanges.All);
-            JavaScriptStringEncoder testEncoder = JavaScriptStringEncoder.UnsafeRelaxedJsonEscaping;
+            JavaScriptEncoder controlEncoder = JavaScriptEncoder.Create(UnicodeRanges.All);
```

Comment:
Nit: perhaps also rename the class and similarly named members to `JavaScriptEncoderTests`?